File: AngularClient/src/app/core/models/CartItem.ts
export class CartItem{
  GameKey?: string;
  GameName?: string;
  GamePrice?: number;
  Quantity?: number;

  constructor(gameKey?: string, gameName?: string, gamePrice?: number, quantity?: number) {
    this.GameKey = gameKey;
    this.GameName = gameName;
    this.GamePrice = gamePrice;
    this.Quantity = quantity;
  }
}

File: GameShop.BLL/DTO/RedisDTOs/CartItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameShop.BLL.DTO.RedisDTOs
{
    public class CartItem
    {
        public string GameKey { get; set; }

        public string GameName { get; set; }

        public decimal GamePrice { get; set; }

        public short Quantity { get; set; } = 1;
    }
}
